src/ontology/mondo-ingest.Makefile

```diff
@@ -442,7 +442,7 @@ lexical-matches: $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv
 #### Lexmatch-SSSOM-compare #######
 ###################################
 # This goal also creates, for all sources: lexmatch/unmapped_%_lex.tsv, lexmatch/unmapped_%_lex_exact.tsv
-lexmatch/README.md: $(SCRIPTSDIR)/lexmatch-sssom-compare.py $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv $(ALL_EXCLUSION_FILES)
+lexmatch/README.md: #$(SCRIPTSDIR)/lexmatch-sssom-compare.py $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv $(ALL_EXCLUSION_FILES)
```

Comment:
Why are the prerequisites commented out of this goal?

Reply:
Oops. Uncommented and re-ran everything!